### DISCUSSION TITLE
Material properties

### DISCUSSION THREAD

**Discussion by Torubara-Oleg** [mfem/mfem#3051] [Q&A] [answered]
Hello everyone!
I am new to FEM. And I am trying to validate linear elasticity problem results. Can anyone help me with material properties? I need to set properties for my mesh material, but I am not quite sure how to do that. I guess that according to examples(example 2) in MFEM we set material properties by Lame parameters(lambda and mu) which can be calculate from Young's modulus and Poisson's ratio. Am I right?

**Comment by BBBBBbruce** [comment, marked as answer]
I am not sure which part your got confused. Below I added how I set the Real2Sim parameters to the material.
I put two materials with same property. 

```
   double Young = 1.0e+8;
   double Poisson = 0.4;

   Lambda = Young * Poisson / (1 + Poisson) / (1 - 2 * Poisson);
   Mu = 0.5 * Young / (1 + Poisson);

   Vector lambda(mesh->attributes.Max());
   lambda = 1.0 * Lambda;
   PWConstCoefficient lambda_func(lambda);
   Vector mu(mesh->attributes.Max());
   mu = 1.0 * Mu;
   PWConstCoefficient mu_func(mu);
```

**Reply by Torubara-Oleg** [reply]
Hi @BBBBBbruce, thank you. Your answer helped me a lot.
